packages/styled-components/src/models/StyledComponent.ts

```diff
@@ -37,6 +38,14 @@ import { DefaultTheme, ThemeContext } from './ThemeProvider';
 
 const identifiers: { [key: string]: number } = {};
 
+const isBrowser = typeof document !== 'undefined';
```

Comment:
Can you switch this to `IS_BROWSER` from constants?

Reply:
Done :D